extensions/blocks/podcast-player/edit.js

```diff
@@ -81,21 +82,30 @@ const PodcastPlayerEdit = ( {
 	const [ editedUrl, setEditedUrl ] = useState( url || '' );
 	const [ isEditing, setIsEditing ] = useState( false );
 	const [ feedData, setFeedData ] = useState( {} );
+	const cancellableFetch = useRef();
 
 	const fetchFeed = useCallback(
 		urlToFetch => {
 			const encodedURL = encodeURIComponent( urlToFetch );
 
-			apiFetch( {
-				path: '/wpcom/v2/podcast-player?url=' + encodedURL,
-			} ).then(
+			cancellableFetch.current = makeCancellable(
+				apiFetch( {
+					path: '/wpcom/v2/podcast-player?url=' + encodedURL,
+				} )
+			);
+
+			cancellableFetch.current.promise.then(
 				data => {
```

Comment:
`isCancel` seems to be present in the `data` object too. Does it make sense to check if the request has been canceled here in the same way when an error happens?

Reply:
Great spot. You are absolutely correct. I've fixed this also.